src/sw-template.js

```diff
@@ -0,0 +1,97 @@
+if ('function' === typeof importScripts) {
+    importScripts(
+      'https://storage.googleapis.com/workbox-cdn/releases/3.5.0/workbox-sw.js'
+    );
+    
+    /* global workbox */
+    if (workbox) {
+      
```

Comment:
nit: extra space here

Reply:
After removing the console log, there no longer is an extra space.  In general, I add a new line proceeding a single or multi-line comment.